### PR TITLE
make lib composer installable

### PR DESCRIPTION
**Pull request by markushausammann**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,11 +1,19 @@
-PHP-DocRaptor
-=============
+#PHP-DocRaptor
 
+PHP-DocRaptor is a simple API wrapper for [DocRaptor.com](http://www.docraptor.com)
```
Comment (janxious) [inline README.markdown]: Change the link to "https://docraptor.com/". Also, periods.

File: README.markdown [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 
-PHP-DocRaptor is a simple consumer class for [DocRaptor.com](http://www.docraptor.com)
-You will need a DocRaptor account before you can use the class, as it requires a valid API key.
+##Installation
 
-###Usage
+This library is PSR-4 autoloading compliant, you can install it via composer. Just require it in your `composer.json` ...
+
+    "require": {
+        "expectedbehavior/php-docraptor": "dev-master"
```
Comment (markushausammann) [inline README.markdown:11]: The version will be updated whenever a release tag is made. For now (testing) it just installs the master.

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "expectedbehavior/php-docraptor",
+    "description": "Official PHP wrapper for the DocRaptor API.",
+    "keywords": ["princexml", "php", "pdf", "xhtml", "xls", "xlsx", "docraptor"],
+    "homepage": "https://github.com/expectedbehavior/php-docraptor",
+    "type": "library",
+    "version": "dev",
```
Comment (janxious) [inline composer.json:7]: I assume I eventually want this to be numeric for semantic versioning purposes?

**Comment by janxious** [comment]
:+1: will merge if you are happy with where this is @markushausammann

File: composer.json (hunk 3 above)
Comment (markushausammann) [inline composer.json:7]: After testing this, we can release a version. It needs to be correct here, as well as in the composer.json and the installation instructions.

**Comment by markushausammann** [comment]
yep, if you could merge and then have it listed on packagist.org that'd be great. I would then install it and push some more PRs. (When installing for development, you can do `composer update --prefer-source` to have composer clone the repo into the vendor folder. Autoloader will be dumped just the same.

**Comment by janxious** [comment]
I think this thing is on? https://packagist.org/packages/expectedbehavior/php-docraptor

**Comment by markushausammann** [comment]
:+1: perfect, please make sure you enable the hook, makes things much easier.

**Comment by markushausammann** [comment]
Now let's see if I did everything right.

**Comment by janxious** [comment]
AFAICT, everything is turned on to full automatic. But this is first time seeing any of the packagist interface. ;)

**Comment by markushausammann** [comment]
Everything seems great:

**Comment by janxious** [comment]
:tada: Cool! Hope this makes your and others' lives easier.

**Comment by janxious** [comment]
And also doesn't make EB look like PHP neanderthals

**Comment by markushausammann** [comment]
Oh, yes, it makes many things easier. To avoid looking like php neanderthals completely a few more PRs are coming... ;)

**Comment by janxious** [comment]
Woo PRs! :+1: 

Also, if you wanted a real-time chat (seemed like you did on the other repo's ticket), I am generally available via Skype: "joel.meador".

**Comment by markushausammann** [comment]
Yeah, for a moment I thought it might help but everything went super smoothly, so for now I guess it's ok, I'll add you anyway in case future PRs create need for extended discussion.

**Pull request merged**